tlsconfig/config.go

```diff
@@ -99,6 +100,11 @@ func (c Config) Parse() (out *tls.Config, err error) {
 		out.MaxVersion = version
 	}
 
+	if out != nil && (out.MinVersion == tls.VersionSSL30 || out.MaxVersion == tls.VersionSSL30) {
+		_, _ = fmt.Fprintln(os.Stderr, "WARNING: SSL 3.0 is depreciated and support for it will be removed soon")
+		return out, nil
```

Comment:
Does this `if` branch need its own return? It looks the same as the one right after the `if statement`.